protocol/message.go

```diff
@@ -124,15 +137,23 @@ type DirectoryProofs struct {
 	STR []*DirSTR
 }
 
+// An STRList response includes a list of signed tree roots
+// STR. A CONIKS auditor returns this DirectoryResponse type upon an
+// AudutingRequest.
+type STRList struct {
```

Comment:
I also liked `ObservedSTRs` a bit better :) but I changed it because of https://github.com/coniks-sys/coniks-go/pull/155/files/4ec8fb73df460e31d2ab3e35da49da2c6fcaaede#diff-2d3b2346117a9f1fc92ea4a92e285e69

This struct will also be used by the directory to return STRs to the auditor, so I wanted to create a more generic data structure.